input/docs/getting-started/upgrade.md

```diff
@@ -3,6 +3,28 @@ Title: Upgrade instructions
 Description: Upgrade instructions between different versions of Cake
 ---
 
+If you are using the [.NET tool as we recommend](/docs/running-builds/runners/dotnet-tool), all you need to do is to update the Cake .NET tool:
+
+```powershell
+# If you are using the .NET tool as a local tool for the repository
+dotnet tool install --local Cake.Tool
+
+# or
+
+# If you are using the .NET tool as a global tool for the user
+dotnet tool install --global Cake.Tool
+```
+
+### Note about Cake.DotNetTool.Module
```

Comment:
Focus the heading to the essential part:

```suggestion
Cake.DotNetTool.Module
```

Reply:
:white_check_mark: